kolibri/core/assets/src/views/tab-link/index.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+
+  <li>
+    <router-link
+      :to="link"
+      class="tab"
+      :class="{ selected: selected, 'tab-icon-title': icon && title }">
```

Comment:
Great point! I was able to use vue-router's `router-link-active` class go get rid of the `selected` prop.